.github/workflows/pr-check.yaml

```
@@ -366,6 +366,54 @@ jobs:
+      - name: Run Smoke E2E tests in Development Mode
+        run: pnpm test:e2e:k8s-sanity
+
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@3585e9575db828022551b4231f165eb59a0e74e3 # v5.6.2
+        if: always() # always run even if the previous step fails
+        with:
+          fail_on_failure: true
+          include_passed: true
+          detailed_summary: true
+          annotate_only: true
+          require_tests:  true
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)